plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -122,6 +125,35 @@ public void setLoginPageUri(String loginPageUri) {
     }
 
     /**
+     * Get session manager implementation class.
```

Comment:
In the rest of getters and setts of this class, setter comes after the getter. Try to follow that convention.